sbnobj/Common/Analysis/classes_def.xml

```diff
@@ -1,8 +1,6 @@
 <lcgdict>
 
-  <class name="anab::TPCChannelInfo" ClassVersion="12">
-   <version ClassVersion="12" checksum="332166564"/>
-  </class>
+  <class name="anab::TPCChannelInfo"/>
```

Comment:
Need a version number here and checksum.  mrb will compute checksum but you should supply the first version number.  You could have left the old version 12 stuff and mrb would have made a version 13